documentation/modules/proc-olm-installing-from-operatorhub-using-console.adoc

```diff
@@ -28,23 +28,26 @@ You must install and deploy the {ProductName} Operator in the `openshift-operato
 
 . Read the information about the Operator and click *Install*. The Create Operator Subscription page opens.
 
-. On the *Create Operator Subscription* page, accept all of the default selections and click *Subscribe*.
+. ({KubePlatform} 4.1 only) On the *Create Operator Subscription* page, accept all of the default selections and click *Subscribe*.
 +
-[NOTE]
-====
-*All namespaces on the cluster (default)* installs the Operator in the default
-*openshift-operators* project and makes the Operator available to all projects
+NOTE:
+{KubePlatform} 4.1 only: *All namespaces on the cluster (default)* installs the Operator in the default *{OperatorNamespace}* project and makes the Operator available to all projects
 in the cluster.
-====
++
+({KubePlatform} 4.2 only) On the *Create Operator Subscription* page, select *A specific namespace on the cluster*, select the *{ProductNamespace}* project, and then click *Subscribe*.
```

Comment:
Need to test this step to ensure it is correct (in particular the selecting the project part)